orcid-web/src/main/webapp/static/javascript/script.js

```diff
@@ -4106,4 +4102,11 @@ function isIndexOf(needle) {
         };
     }
     return indexOf.call(this, needle);
-};
\ No newline at end of file
+};
+
+function checkIfCookiesEnabled() {
+	OrcidCookie.setCookie("cookieTest", "test", 1);
```

Comment:
@nitw-shobhit  Let's make this a function of OrcidCookie so you have to call OrcidCookie.checkIfCookiesEnabled(). Looking good!